sources/Application/Instruments/MidiInstrument.h

```diff
@@ -61,6 +61,7 @@ class MidiInstrument : public I_Instrument {
   bool playing_;
   bool retrig_;
   int retrigLoop_;
+  char velocity_;
```

Comment:
set default value!

Reply:
It's set from the instrument default value but good point! always safer to set a default value on definition.